examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "experimental-offchain-zkapp-storage",
+  "version": "0.1.1",
+  "description": "",
+  "author": "",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/o1-labs/docs2/tree/main/examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage"
+  },
+  "homepage": "https://github.com/o1-labs/docs2/tree/main/examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage#README.md",
+  "bugs": {
+    "url": "https://github.com/o1-labs/docs2"
+  },
+  "keywords": [
+    "mina-zkapp",
+    "mina-zk-app",
+    "mina-dapp",
+    "zkapp"
+  ],
+  "type": "module",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "generate-key": "node scripts/generate_private_key.js",
+    "server": "npm run build && node --enable-source-maps build/src/storageServer.js"
```

Comment:
```suggestion
    "server": "npm run build && node --enable-source-maps build/src/server/storageServer.js"
```

Reply:
Similar comment I made in the readme. 

It looks like this script is broken.  I received this error. 
```
cannot find module 'docs2/tutorial-6/docs2/examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/build/src/storageServer.js'
```

The current package.json script is trying to access the wrong file path.

```
"server": "npm run build && node --enable-source-maps build/src/storageServer.js"
```
 This worked for me.
 
     "server": "npm run build && node --enable-source-maps build/src/server/storageServer.js"